app/templates/components/wikia-ui-components/wikia-card-list.hbs

```diff
@@ -14,5 +14,8 @@
         <div class="wikia-card__snippet wds-font-size-xs">{{item.snippet}}</div>
       {{/link-to}}
     </li>
+    {{#if (equal key 1)}}
+      {{post-search-results query=query seeMore=true isInternal=true}}
+    {{/if}}
```

Comment:
`wikia-card-list` component is used also in file pages. Is it ok to show `post-search-results` there as well?

Reply:
Good catch, I'll check up with Product

Reply:
Fixed in 2b2328f